src/theme/book.js

```diff
@@ -468,18 +481,28 @@ function playpen_text(playpen) {
 
         switch (e.key) {
             case 'ArrowRight':
+            case 'j':
+                e.preventDefault();
+                var previousButton = document.querySelector('.nav-chapters.previous');
+                if (previousButton) {
+                    window.location.href = previousButton.href;
+                }
+                break;
+            case 'ArrowLeft':
+            case 'k':
                 e.preventDefault();
                 var nextButton = document.querySelector('.nav-chapters.next');
                 if (nextButton) {
                     window.location.href = nextButton.href;
                 }
                 break;
-            case 'ArrowLeft':
+            case 'h':
                 e.preventDefault();
-                var previousButton = document.querySelector('.nav-chapters.previous');
-                if (previousButton) {
-                    window.location.href = previousButton.href;
-                }
+                // TODO: add code
+                break;
+            case 'l':
+                e.preventDefault();
+                // TODO: add code
```

Comment:
TODO?

Reply:
seems that the author abandoned this PR, I created #1408 to follow up on this.